personality/argus/pep.pan

```diff
@@ -1,200 +1,251 @@
+  contents = "#!/bin/bash\n";
+  contents = contents + '###############################################################################' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#       Copyright (c) Members of the EGEE Collaboration. 2004' + "\n";
+  contents = contents + '#       See http://eu-egee.org/partners/ for details on the copyright holders' + "\n";
+  contents = contents + '#       For license conditions see the license file or http://eu-egee.org/license.html' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#   Startup script for PEP daemon server' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#   chkconfig: 345 97 97' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#   description:  PEP daemon server startup script' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#   processname: pepd' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#   Author(s): Maria Alandes Pradillo <yaim-contact@cern.ch>' + "\n";
+  contents = contents + '#              Valery Tschopp <argus-support@cern.ch>' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#   Version: V1.2' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '#   Date: 12/04/2010' + "\n";
+  contents = contents + '###############################################################################' + "\n";
+  contents = contents + 'PEP_HOME=' + PEP_HOME + "\n";
+  contents = contents + 'export PEP_HOME' + "\n";
+  contents = contents + "\n";
+  contents = contents + 'if [ `id -u` -ne 0 ]; then' + "\n";
+  contents = contents + '    echo "You need root privileges to run this script"' + "\n";
+  contents = contents + '    exit 1' + "\n";
+  contents = contents + 'fi ' + "\n";
+  contents = contents + "\n";
+  contents = contents + 'case "$1" in' + "\n";
+  contents = contents + '    start)' + "\n";
+  contents = contents + '        ' + PEP_HOME + '/sbin/pepdctl.sh start' + "\n";
+  contents = contents + '        ;; ' + "\n";
+  contents = contents + '    stop)' + "\n";
+  contents = contents + '        ' + PEP_HOME + '/sbin/pepdctl.sh stop' + "\n";
+  contents = contents + '        ;;' + "\n";
+  contents = contents + '    status)' + "\n";
+  contents = contents + '        ' + PEP_HOME + '/sbin/pepdctl.sh status' + "\n";
+  contents = contents + '        ;;' + "\n";
+  contents = contents + '    clearcache)' + "\n";
+  contents = contents + '        ' + PEP_HOME + '/sbin/pepdctl.sh clearResponseCache' + "\n";
+  contents = contents + '        ;;' + "\n";
+  contents = contents + '    *)' + "\n";
+  contents = contents + '        echo "Usage: $0 {start|stop|status|clearcache}"' + "\n";
+  contents = contents + '        exit 1' + "\n";
+  contents = contents + '        ;;' + "\n";
+  contents = contents + 'esac' + "\n";
+  contents = contents + "\n";
+  contents = contents + 'exit $?' + "\n";
+
+  contents;
```

Comment:
replace the interpolated data with
```
PEP_HOME=/some/reasonable/default
if [ -f ]; then
    . /etc/sysconfig/pepd
fi
export PEP_HOME
```

and set the `PEP_HOME` with ncm-sysconfig.
(and use `$PEP_HOME` in the script itself)
add the init script as file with file_contents and filecopy (or better yet, an rpm)